src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,20 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../Types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    className={`
+      TodoInfo
+      ${todo.completed && 'TodoInfo--completed'}`}
+    data-id={todo.id}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    {todo.user && <UserInfo user={todo.user} />}
```

Comment:
Destructure the todo in this component, since you're using more than one of its keys